cmd/configure/main.go

```diff
@@ -1,89 +1,21 @@
 package main
 
 import (
-	"errors"
-	"fmt"
-	"io/ioutil"
 	"log"
-	"os"
-	"path/filepath"
-	"regexp"
-	"text/template"
+
+	"github.com/paketo-buildpacks/nginx/cmd/configure/internal"
 )
 
 func main() {
 	log.SetFlags(0)
 
-	mainConf := os.Args[1]
-	body, err := ioutil.ReadFile(mainConf)
-	if err != nil {
-		log.Fatalf("Could not read config file (%s): %s", mainConf, err)
-	}
+	err := internal.Run(
+		"/workspace/nginx.conf",
+		"/workspace/modules",
```

Comment:
The `/workspace` directory here can be referenced as `os.Getenv("CNB_APP_DIR")`.

You should check out the [exec.d code in `npm-install`](https://github.com/paketo-buildpacks/npm-install/blob/main/cmd/setup-symlinks/main.go) for some further pointers on how to do this without hardcoding these values.

Reply:
@ryanmoran `os.Getenv("CNB_APP_DIR")` returns an empty string instead of `/workspace` in this case.

Reply:
@ryanmoran `os.Getenv("CNB_APP_DIR")` returns an empty string instead of `/workspace` in this case.

Reply:
hmm. that makes me wonder how the `npm-install` buildpack is working then. @ForestEckhardt thoughts?

Reply:
I cannot figure heads or tails on that one. I would check to make sure you have the latest `pack` version installed for testing be deleting `.bin`.

If you still see that the env var is empty let me know how you are seeing the value of the env var and I will do some testing of my own.